### ISSUE TITLE
LibreQuake weapons and enemies don't load in Quake Enhanced.

### ISSUE THREAD

**Issue by eggmanstudio** [lavenderdotpet/LibreQuake#245]
LibreQuake works fine in Quake Enhanced, I have found - except the enemy and weapon models remain those from Quake as opposed to loading LibreQuake's. Might the remaster be trying to do something a bit differently to what LibreQuake expects?

**Comment by MotoLegacy** [comment]
kexQuake patches the model paths to use higher fidelity IQM meshes if you have that setting turned on in the graphics menu. Not sure if that's exposed or overwritable. A lot of those enhancements cannot be touched by mods.

**Comment by lavenderdotpet** [comment]
might be fixable by having our own md5s but currently not ready for that

**Comment by lavenderdotpet** [comment]
> WIth the latest update, disabling Enhanced Models in the game settings now correctly displays the LibreQuake models - was this an intended fix? In any case, whether intentional or not, this has now been fixed!

yeah again we dont have md5 models so disabling enhanced models would of always been the fix

md5 exporters are not at the point where we could easily support md5 so its not happening as of yet
if do end up supporting md5

the community is currently working on getting most engines to support md3 as an enhancement model so maybe we will just go that route and not worry about kexquake support
